### DISCUSSION TITLE
Confused Calibration 1

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#3657] [General]
I started a new sensor a few hours ago.  After warm up,, I didn't have ready access to the code, so I entered 0000 to enter a manual calibration.  I entered the calibration, but the transmitter did not accept it. Instead, it returned with "Confused Calibration 1" error, and in the event logs "Encoutered backfill data we don't recognize".    Now, the display reads "not getting any readings".  Is there anything I can do to fix this error?

**Comment by Navid200** [comment]
When confused calibration occurs, the only option left is to either wait for it to recover, or restart.
That's why I removed the option of starting in the no-code mode from my guide.  There is a risk of getting confused calibration.
